javascript/app.js

```diff
@@ -42,11 +42,15 @@ FinancialFreedom.directive('autofocus', ['$timeout', function($timeout) {
   }
 }]);
 
-FinancialFreedom.controller('bodyController', ['$scope', '$location',  function($scope, $location) {
+FinancialFreedom.controller('bodyController', ['$scope', '$location', '$window', function($scope, $location, $window) {
     $location.path("/");
     $scope.isActive = function(route) {
         return route == $location.path();
     };
+
+    $scope.$on('$routeChangeSuccess', function(event) {
+        $window.ga('send', 'pageview', { page: $location.url() });
+    });

```

Comment:
good call—see latest version. it's even easier—the service itself can watch for those events, so we don't need any code at all in the controller, besides injecting the new service.